requirements.txt

```diff
@@ -1,6 +1,103 @@
 #
-# This file is autogenerated by pip-compile with python 3.10
```

Comment:
We can probably revert the changes to the `requirements.txt` file as well, since it's an artifact of the `requirements.in` file.

However! This would be a good time to check if items like the `setuptools` pinning will happen with the `requirements.in` file being empty :grin: